2018-11-28 12:36:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "TABLE1VALUES" not found; SQL statement:
INSERT INTO TABLE1VALUES(r.nextGaussian()*100+500) [42102-197]
2018-11-28 12:36:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Function "NEXTGAUSSIAN" not found; SQL statement:
INSERT INTO TABLE1 VALUES(r.nextGaussian()*100+500) [90022-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.readJavaFunction(Parser.java:2699)
	at org.h2.command.Parser.readFunction(Parser.java:2756)
	at org.h2.command.Parser.readTermObjectDot(Parser.java:2972)
	at org.h2.command.Parser.readTerm(Parser.java:3095)
	at org.h2.command.Parser.readFactor(Parser.java:2587)
	at org.h2.command.Parser.readSum(Parser.java:2574)
	at org.h2.command.Parser.readConcat(Parser.java:2544)
	at org.h2.command.Parser.readCondition(Parser.java:2370)
	at org.h2.command.Parser.readAnd(Parser.java:2342)
	at org.h2.command.Parser.readExpression(Parser.java:2334)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1280)
	at org.h2.command.Parser.parseInsert(Parser.java:1222)
	at org.h2.command.Parser.parsePrepared(Parser.java:431)
	at org.h2.command.Parser.parse(Parser.java:335)
	at org.h2.command.Parser.parse(Parser.java:307)
	at org.h2.command.Parser.prepareCommand(Parser.java:278)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at NormalDistribution.MainClass.main(MainClass.java:29)
2018-11-28 12:38:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO TABLE1 VALUES(7.896819347262335500) [21002-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:306)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at NormalDistribution.MainClass.main(MainClass.java:29)
2018-11-28 12:38:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO TABLE1 VALUES(-90.20431420694814500) [21002-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:306)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at NormalDistribution.MainClass.main(MainClass.java:28)
2018-11-28 12:39:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO TABLE1 VALUES(-45.4613204327221500) [21002-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:306)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at NormalDistribution.MainClass.main(MainClass.java:28)
2018-11-28 12:40:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Method is only allowed for a query. Use execute or executeUpdate instead of executeQuery; SQL statement:
INSERT INTO TABLE1 (NDNumber)VALUES(32.06377604041404500) [90002-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.Prepared.query(Prepared.java:219)
	at org.h2.command.CommandContainer.query(CommandContainer.java:114)
	at org.h2.command.Command.executeQuery(Command.java:202)
	at org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:85)
	at NormalDistribution.MainClass.main(MainClass.java:28)
